src/components/approvallist/ApprovalHeader.tsx

```diff
@@ -13,50 +14,65 @@ const StyledFirstRow = styled.div`
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
+  margin-bottom: 16px;
+  padding: 0px 16px;
 `;
 
 export const ApprovalHeader = observer(() => {
   const { uiStore } = useContext(StoreContext);
+  const [approvalDialogOpen, setApprovalDialogOpen] = useState(false);
+  const theme = useTheme();
 
   return (
     <>
+      {approvalDialogOpen && (
+        <div style={{ position: 'absolute', top: '0px', height: '100%', width: '100%' }}>
+          <ApprovalDialog onCancel={() => setApprovalDialogOpen(false)} />
+        </div>
+      )}
       <StyledFirstRow>
-        <Title size="md">Approvals</Title>
-        <Settings />
+        <Typography variant="h4" fontWeight={700}>
+          Approvals
+        </Typography>
+        <Box display="flex" flexDirection="row" gap={2}>
+          <Settings />
+          <Button
+            variant={uiStore.selectedApprovals.length === 0 ? 'outlined' : 'contained'}
+            disabled={uiStore.selectedApprovals.length === 0}
+            size="large"
+            onClick={() => setApprovalDialogOpen(true)}
+          >
+            Edit Approvals
+          </Button>
+        </Box>
       </StyledFirstRow>
+      <Divider />
       <ColumnGrid
         style={{
-          paddingLeft: 22,
-          paddingRight: 44,
           minHeight: 48,
-          borderBottom: '2px solid #E8E7E6',
-          width: 'inherit',
+          borderBottom: `1px solid ${theme.palette.border.main}`,
```

Comment:
You can remove reliance on `useTheme` by using a callback here.

```suggestion
          borderBottom: (theme) => `1px solid ${theme.palette.border.main}`,
```